packages/create-mc-app/src/process-options.ts

```diff
@@ -72,6 +74,7 @@ async function processOptions(
   const templateName = options.template;
 
   // Validate options
+  throwIfApplicationTypeIsNotSupported(applicationType);
```

Comment:
Check that the received value for the new parameter is valid: `custom-application`, `custom-view`.